README.md

```diff
@@ -0,0 +1,39 @@
+This is a browser-based tool used to assist in resolving duplicate urls in the PDAP Agencies Airtable.
+
+Pulls data from [PDAP Agencies Airtable](https://airtable.com/app473MWXVJVaD7Es/shr43ihbyM8DDkKx4/tblpnd3ei5SlibcCX).
+
+To utilize this tool, either utilize the PDAP Interface (not currently completed) or download a csv of the Agencies data and run initial_data_load.py to load the data into the database.
+
+```bash
+python data_processing/initial_data_load.py --csv input.csv
+```
+
+And then run pipeline.py to pull data from the web and update the database.
+
+```bash
+python data_processing/pipeline.py
```

Comment:
i had to add `google-api-python-client` to the requirements.txt here. I struggled a bit with polars install because of cache stuff—always worth telling people to use a `venv`, imo, because the install is more predictable

Reply:
Almost there...I can't resolve this:

```
    from data_processing.DatabaseManager import DatabaseManager
ModuleNotFoundError: No module named 'data_processing'
```

Reply:
@josh-chamberlain I attempted to address this in the most recent version of my commits. Also other details, which I'll describe in my comment below.

Reply:
I did get a better error: 

```
RuntimeError: Custom search API key and CSE ID must be set in .env file
```

and I was able to resolve it with the README.